eng/Build.props

```diff
@@ -23,6 +23,9 @@
     <DependencyPackageProjects Include="$(RepoRoot)src\referencePackages\src\**\Microsoft.Bcl.AsyncInterfaces.6.0.0.csproj" />
     <DependencyPackageProjects Include="$(RepoRoot)src\referencePackages\src\**\System.Text.Encodings.Web.6.0.0.csproj" />
     <DependencyPackageProjects Include="$(RepoRoot)src\referencePackages\src\**\System.Text.Json.6.0.0.csproj" />
+
+    <DependencyPackageProjects Include="$(RepoRoot)src\referencePackages\src\**\System.Memory.4.5.4.csproj" />
```

Comment:
This shouldn't be needed since this package is already included.

Reply:
Thanks! Right - forgot to remove `System.Memory` entry - will update this.

Reply:
Fixed with commit: https://github.com/dotnet/source-build-reference-packages/pull/591/commits/0c8e2e7bb94ddf68dbc1300a772a660b9376417b